developer_manual/core/theming.rst

```diff
@@ -1,166 +1,183 @@
+----------------------------------
+
+To override the default style sheet, create a new CSS style sheet in your theme, in the theme’s ``css`` directory, called ``styles.css``.
+
+To override the background in the login page and the blue header bar on the main navigation page, visible once you log in to ownCloud, you need to override the CSS ``body-login`` element , which you can see the default definition of below. 
+It implements a `CSS gradient`_ with support for a range of browsers, both old and new. 
+
+.. note: 
+   Not all browsers support CSS gradients.
+   To find out which do, and which don’t, check `the guide on Can I Use`_.
 
 .. code-block:: css
 
   /* HEADERS */
   ...
   body-login {
-    background: #1d2d42; /* Old browsers */
-    background: -moz-linear-gradient(top, #33537a 0%, #1d2d42  100%); /* FF3.6+ */
-    background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#F1B3A4), color-stop(100%,#1d2d42)); /* Chrome,Safari4+ */
-    background: -webkit-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* Chrome10+,Safari5.1+ */
-    background: -o-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* Opera11.10+ */
-    background: -ms-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* IE10+ */
-    background: linear-gradient(top, #33537a 0%,#1d2d42 100%); /* W3C */
+   /* Old browsers */
+   background: #745bca;
+   /* FF3.6+ */
+   background: -moz-linear-gradient(top, #947bea 0%, #745bca 100%);
+   /* Chrome,Safari4+ */
+   background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #947bea), color-stop(100%, #745bca));
+   /* Chrome10+,Safari5.1+ */
+   background: -webkit-linear-gradient(top, #947bea 0%, #745bca 100%);
+   /* Opera11.10+ */
+   background: -o-linear-gradient(top, #947bea 0%, #745bca 100%);
+   /* IE10+ */
+   background: -ms-linear-gradient(top, #947bea 0%, #745bca 100%);
+   /* W3C */
+   background: linear-gradient(top, #947bea 0%, #745bca 100%);
+   /* IE6-9 */
+   filter: progid: DXImageTransform.Microsoft.gradient( startColorstr='#947bea', endColorstr='#745bca', GradientType=0 );
   }
 
-The different background-assignments indicate the headers for a lot of different browser types. What you most likely want to do is change the #35537a (lighter blue) and #ld2d42 (dark blue) color to the colours of our choice. In some older and other browsers, there is just one color, but in the rest showing gradients is possible.
-The login page background is a horizontal gradient. The first hex number, #35537a, is the top color of the gradient at the login screen. The second hex number, #ld2d42 is the bottom color of the gradient at the login screen.
-The gradient in top of the normal view after login is also defined by these CSS-settings, so that they take effect in logged in situation as well.
-Change these colors to the hex color of your choice:
-As usual:
+When changing the gradient what you most likely want to do is change ``#35537a`` and ``#ld2d42`` to the colors of your choice. 
```

Comment:
I don't mind little excursions like this, but do we want to educate our users in css? They should know css, and if they don't, there are better resources to learn it.

But like i said, it's not too bad.